sovtokenfees/sovtokenfees/test/test_set_get_fees.py

```diff
@@ -125,7 +125,10 @@ def test_set_fees_with_stewards(helpers):
     """
     fees = {NYM: 1}
     fees_request = helpers.request.set_fees(fees)
-    fees_request.signatures.popitem()
+    sigs = [(k, v) for k, v in fees_request.signatures.items()]
```

Comment:
Why do we need to do this?

Reply:
Because `popitem` removes random field from `dict`